sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/eventhubs/properties/AzureEventHubsProperties.java

```diff
@@ -228,6 +229,65 @@ public BlobCheckpointStore getCheckpointStore() {
             return checkpointStore;
         }
 
+        /**
+         * The starting position from which to consume events.
+         */
+        public static class StartPosition implements EventProcessorClientProperties.StartPosition {
+
+            /**
+             * The offset of the event within that partition. String keyword, "earliest" and "latest" (case-insensitive),
+             * are reserved for specifying the start and end of the partition. Other provided value will be cast to Long.
+             */
+            private String offset;
+            /**
+             * The sequence number of the event within that partition.
+             */
+            private Long sequenceNumber;
+            /**
+             * The event enqueued after the requested enqueuedDateTime becomes the current position.
```

Comment:
This member is an Instant, thus is the javadoc should be "The event enqueued time after the requested enqueuedDateTime becomes the current position."